File: NutritionControl.DTO/Models/ProfileDto.cs
namespace NutritionControl.DTO.Models
{
    public class ProfileDto
    { 
        public decimal? Height { get; set; }
        public decimal? Weight { get; set; }
        public int? Age { get; set; }
        public decimal? BMI { get; set; }
        public decimal? GoalWeight { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string Email { get; set; }
        public string Gender { get; set; }
    }
}

File: NutritionControl.Client/client-app/src/app/models/profiledto.ts
export class ProfileDto {
  height: number;
  weight: number;
  age: number;
  bmi: number;
  goalWeight: number;
  name: string;
  surname: string;
  email: string;
  gender: string;
}

export class PasswordChangeRequest{
  oldPassword: string;
  newPassword: string;
  confirmPassword: string;
}